Add E3D BigBox Dual Configuration Example

RainMotorsports opened the pull request:
### Requirements

Requires this PR adding support for this printer using PT100 amplifiers over the Rumba's EXP3 header from the factory. https://github.com/MarlinFirmware/Marlin/pull/27897

### Description

This configuration is for the E3D BigBox Dual in stock configuration. Tested on my personal machine with values checked against the original Marlin 1.1.0 RC configurations.

RainMotorsports committed:
Add E3D BigBox Dual Configuration.h to Examples

RainMotorsports committed:
Add E3D BigBox Dual Configuration_adv.h to Examples

RainMotorsports committed:
Add README.md for BigBox Dual

thisiskeithb commented on config/examples/E3D/BigBox Dual/README.md:
```diff
@@ -0,0 +1,20 @@
+# Microstepping
+The Y axis should be jumpered to 8 microsteps with the stock geared setup with 0.9 degree stepper. 16 microsteps even at low speeds would overrun the RUMBA. 1.8 degree steppers can run 16 microsteps and there would be no change to the steps per mm in this configuration.
+
+#Probe
```
This is missing a space after `#` so it doesn’t render as a heading.

RainMotorsports replied:
Fixed, since there was a mistake there I went ahead and fixed a non critical issue in the config I had noticed after running through a diff on my end. I had set the Z max to 50 when it was 100 in the original firmware. Should all be good to go pending the PR for the rumba pins.

RainMotorsports committed:
Fixed Heading mistake in README.md

RainMotorsports committed:
Update Default Max Acceleration in Configuration.h for BigBox Dual

While fixing the readme I fixed this mistaken value 50 for the actual default of 100.

thinkyhead committed:
Merge branch 'import-2.1.x' into pr/1144

thinkyhead committed:
clean up
